cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -668,6 +689,7 @@ def _add_lambda_cleanup(self, policy_statements, build_tags):
             function_name=lambda_cleanup.attr_arn,
             source_arn=Fn.ref("BuildNotificationTopic"),
         )
+        lambda_cleanup.add_depends_on(lambda_log)
```

Comment:
Because the log must be created in advance, otherwise you can get a failure for "log group already exists" error.
> the lambda cleanup function should be deleted before lambda log?

yes, like the lambda cleanup function is deleted before the lambda execution role